src/ve-app/pane-center/document.component.ts

```diff
@@ -373,15 +367,15 @@ class FullDocumentController implements IComponentController {
                 refType: this.mmsRef.type,
             }
 
-            this.eventSvc.$broadcast<veAppEvents.elementSelectedData>('view.selected', data)
-            if (viewId === this.processed) return
-            this.processed = viewId
+            this.eventSvc.$broadcast<veAppEvents.elementSelectedData>('element.selected', data)
+            //if (viewId === this.processed) return
+            //this.processed = viewId
             this.fullDocumentApi.handleClickOnBranch(viewId, () => {
-                this.$location.hash(viewId)
+                this.$location.hash(viewId) //this is causing a state change, transition will be canceled
```

Comment:
Yeah this isn't relevant anymore because `viewId` has been moved to be a query parameter, we should try to remove it